feat: Add sorting and filtering releases docs (#3861)
getsentry/sentry-docs: src/docs/product/sentry-basics/search/searchable-properties.mdx

| message | Restrict results to events with a matching message. |
| environment | Restrict results to events tagged with the matching environment. |
| release | Restrict results to events tagged with the matching release. You can create a token with an exact match on the version of a release, or `release:latest` to pick the most recent release. |

> | release.package<br /> release.version<br /> release.build | Restrict results to events tagged with the matching semantic version. Supports ranges and wildcards. For more information, see [Sorting and Filtering Releases](/product/releases/usage/sorting-filtering/). |

Reviewer: ```suggestion
| release.package<br /> release.version<br /> release.build | Restrict results to events tagged with the matching semantic version. Supports ranges and wildcards. Learn more in [Sorting & Filtering Releases](/product/releases/usage/sorting-filtering/). |
```
